ChromeExtension/js/background.js

```diff
@@ -1,22 +1,4 @@
-chrome.runtime.onInstalled.addListener(function (details) {

```

Comment:
this is moved to the serviceworker.

some functions are kept here intentionally (ones that still need access to the DOM)

https://developer.chrome.com/docs/extensions/develop/migrate/to-service-workers#changes-over-bg